README.md

```diff
@@ -428,6 +437,203 @@ HTTP/1.1 200 OK
+
+#### Committing Cursors
+Cursors can be committed by posting to `/subscriptions/{subscriptionId}/cursors`. Example:
+```sh
+curl -v -X POST \
+    -H "Content-type: application/json" \
+    -H "X-Nakadi-StreamId: ae1e39c3-219d-49a9-b444-777b4b03e84c" \    
+    "http://localhost:8080/subscriptions/038fc871-1d2c-4e2e-aa29-1579e8f2e71f/cursors" \
+    -d '{
+      "items": [
+        {
+          "partition": "5",
+          "offset": "543",
+          "event_type": "order.ORDER_RECEIVED",
+          "cursor_token": "b75c3102-98a4-4385-a5fd-b96f1d7872f2"
+        },
+        {
+          "partition": "2",
+          "offset": "923",
+          "event_type": "order.ORDER_RECEIVED",
+          "cursor_token": "a28568a9-1ca0-4d9f-b519-dd6dd4b7a610"
+        }
+      ]
+    }'
+```
+Please be aware that `X-Nakadi-StreamId` header is required when doing a commit.
+The value should be the same as you get in `X-Nakadi-StreamId` header when opening a stream of events. 
+Also, each client can commit only the batches that were sent to him.
+
+The possible success answers from commit endpoints are:
+
+- `204`: cursors were successfully committed and offset was increased;
+- `200`: cursors were committed but at least one of the cursors didn't increase the offset as it was 
+less or equal to already committed one. In a case of this response code user will get a json in a 
+response body with a list of cursors and the results of their commits.
+
+#### Checking Current Position
+To see what is current position of subscription it's possible to run the request:
+```sh
+curl -v -X GET "http://localhost:8080/subscriptions/038fc871-1d2c-4e2e-aa29-1579e8f2e71f/cursors"
+```
+The response will be a list of current cursors that reflect the last committed offsets.
+
+#### Subscription Statistics
+To get statistics of subscription the folowing request should be used:
+```sh
+curl -v -X GET "http://localhost:8080/subscriptions/038fc871-1d2c-4e2e-aa29-1579e8f2e71f/stats"
+```
+
+The output will contain the statistics for all partitions of the stream. Like this:
+```sh
+{
+  "items": [
+    {
+      "event_type": "order.ORDER_RECEIVED",
+      "partitions": [
+        {
+          "partition": "0",
+          "state": "reassigning",
+          "unconsumed_events": 2115,
+          "client_id": "b75c3102-98a4-4385-a5fd-b96f1d7872f2"
+        },
+        {
+          "partition": "1",
+          "state": "assigned",
+          "unconsumed_events": 1029,
+          "client_id": "ae1e39c3-219d-49a9-b444-777b4b03e84c"
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### Deleting Subscription
+To delete subscription one should run DELETE request on /subscriptions/{subscriptionId}
+```sh
+curl -v -X DELETE "http://localhost:8080/subscriptions/038fc871-1d2c-4e2e-aa29-1579e8f2e71f"
+```
+
+#### Getting and Listing Subscriptions
+Getting single subscription by id:
+```sh
+curl -v -X GET "http://localhost:8080/subscriptions/038fc871-1d2c-4e2e-aa29-1579e8f2e71f"
+```
```

Comment:
@antban so for me there are two ways:
1) Specify the example answer for every endpoint
2) Skip the example answer for endpoints where the answer is obvious